backend/controllers/prisonerProfile/prisonerAlerts.ts

```diff
@@ -14,7 +14,7 @@ export default ({
   async (req, res) => {
     const { offenderNo } = req.params
 
-    const { context, overrideAccess } = await getContext({
```

Comment:
It's not as simple as allowing POMs access to DPS anymore - we need other details from Restricted Patients